Update whats-new.mdx (#1679)
grommet/hpe-design-system: aries-site/src/pages/whats-new.mdx

- Added [download links for MetricHPEXS](/foundation/typography#fonts-for-offline-usage) for offline usage.
- Enhanced filtering examples to demonstrate [Filtering with a table containing selectable rows](/templates/filtering#filtering-with-selectable-results).
- Added guidance for [setting the width of a table](/components/table#setting-the-width-of-the-table).

> - Added a [checked checkbox variant to Figma for Table headers]( https://www.figma.com/file/XEoIWIlLfcARRIFhkpFCch/HPE-Table?node-id=2390%3A13994).

Reviewer: ```suggestion
- Added a [CheckBox variant of "checked" to Figma for Table headers]( https://www.figma.com/file/XEoIWIlLfcARRIFhkpFCch/HPE-Table?node-id=2390%3A13994).
```

> - Aligned the [checkbox component in Figma](https://www.figma.com/file/7Mm1xDBTOtPHqggEVpaD2N/HPE-Checkbox-Component?node-id=1315%3A1) to the Design System site to include a border when on hover.

Reviewer: ```suggestion
- Aligned the [CheckBox component in Figma](https://www.figma.com/file/7Mm1xDBTOtPHqggEVpaD2N/HPE-Checkbox-Component?node-id=1315%3A1) with the Design System site to include a border on hover.
```

- Added guiance and examples for [FileInput](/components/fileinput)
- Added `kind="toolbar"` to button in grommet-theme-hpe for use in Filter contexts. See how this button is used in [Filtering examples](/templates/filtering).

> - Replaced Anchor Button with Defualt Button on when clearing filters on [Filtering examples on Design System site](https://design-system.hpe.design/templates/filtering) and [Figma](https://www.figma.com/file/EA0OZFdocKVcQ55c7Jainb/HPE-Filter-Templates?node-id=1%3A3738).

Reviewer: ```suggestion
- Updated filtering pattern to use default Button instead of Anchor for "Clear Filters" control. View the [Filtering examples on Design System site](https://design-system.hpe.design/templates/filtering) and [Figma](https://www.figma.com/file/EA0OZFdocKVcQ55c7Jainb/HPE-Filter-Templates?node-id=1%3A3738).
```

- Added guidance for [Spinner](/components/spinner)
- Improved accessibility for Select by [allowing callers to specify an `a11yTitle`](https://github.com/grommet/grommet/pull/5234).
- Improved the Design System site experience for keyboard navigators and screen reader users by [adding SkipLinks](https://github.com/grommet/hpe-design-system/pull/1606). 

> - Completed gathering research and screen shots of the different types of [Page Layouts](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/App-Page-Layouts?node-id=881%3A3952).

Reviewer: ```suggestion
- Completed research and catalog of screen shots of different types of [Page Layouts](https://www.figma.com/file/4rdBkUlHd5MCVq3hvUOXHd/App-Page-Layouts?node-id=881%3A3952).
```

## April 19-23, 2021

- Added [examples of text sizes 3xl, 4xl, 5xl, 6xl](/foundation/typography#text)

> - Added [new text sizes to Figma](https://www.figma.com/file/oJhw3JqMemtbwWjlLPWW5O/HPE-Typography-Styles?node-id=149%3A2).

Reviewer: ```suggestion
- Added [text sizes of 3xl, 4xl, 5xl, and 6xl to Figma](https://www.figma.com/file/oJhw3JqMemtbwWjlLPWW5O/HPE-Typography-Styles?node-id=149%3A2).
```

## March 22-26, 2021

- Added theming for [Pagination](https://github.com/grommet/grommet-theme-hpe/pull/174) and [Spinner](https://github.com/grommet/grommet-theme-hpe/pull/176) to grommet-theme-hpe

> - Ideation on [FileInput](https://www.figma.com/file/8HwAiiSDloOTjUxqvCTypq/HPE-FileInput-Component?node-id=0%3A1) completed.

Reviewer: ```suggestion
- Completed ideation on [FileInput](https://www.figma.com/file/8HwAiiSDloOTjUxqvCTypq/HPE-FileInput-Component?node-id=0%3A1).
```

Contribution from the Grommet community!
- Resolved lag in hover styling https://github.com/grommet/grommet/issues/4881.

### Tabs

> - Resolved [spacing issue between icons within the tabs in Figma](https://www.figma.com/file/Kp4dWyhUTnKIJ1Cg5CoL9o/HPE-Tabs-Component?node-id=0%3A1).

Reviewer: ```suggestion
- Resolved [spacing issue between icons within Tabs in Figma](https://www.figma.com/file/Kp4dWyhUTnKIJ1Cg5CoL9o/HPE-Tabs-Component?node-id=0%3A1).
```

- Fixed issue with [Select with search](https://github.com/grommet/grommet/issues/4829)
- Enhance Layer theming to [allow styling border radii](https://github.com/grommet/grommet/issues/4805).
- Completed [Ideation phase for What's New page](https://github.com/grommet/hpe-design-system/issues/822) with recommended design for implementation.

> - Research completed on [Principles and Values](https://www.figma.com/file/8jLGXR0nBgFeuy9cdQUxWH/Principles-and-Values?node-id=0%3A1) to create our own.

Reviewer: ```suggestion
- Completed research on [Principles and Values](https://www.figma.com/file/8jLGXR0nBgFeuy9cdQUxWH/Principles-and-Values?node-id=0%3A1) to enhancements to our own.
```
